_integration-test/run.sh

```diff
@@ -41,10 +45,10 @@ echo "${_endgroup}"
 echo "${_group}Starting Sentry for tests ..."
 # Disable beacon for e2e tests
 echo 'SENTRY_BEACON=False' >>$SENTRY_CONFIG_PY
-echo y | $dcr web createuser --force-update --superuser --email $TEST_USER --password $TEST_PASS
```

Comment:
docker run is slower than just an exec into a running container

Reply:
It is but it is also intentional to keep the run separate from the main web process.

That said for a one-off thing like this, I think using `exec` is a good compromise if it saves a notable amount of time. I'd just want it documented with a brief comment.